samples/abn-sample/backend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Get Iter8
+FROM golang:1.18-buster as builder
```

Comment:
Dockerfile says 1.18. Go mod file says 1.17. Is this an issue? Should this be consistent?

Reply:
agreed. done. verified match in `frontend/go` as well.